packages/visx-shape/src/shapes/Polygon.tsx

```diff
@@ -39,10 +39,12 @@ export const getPoints = ({
 
 export type PolygonProps = {
   /** Number of polygon sides. */
-  sides: number;
+  sides?: number;
```

Comment:
Above we are getting TS errors because `sides` and `size` are now optional in `PolygonProps`. It may work if you update the `getPoint` signature type to something like this 

```tsx
... & NonNullable<Pick<PolygonProps, 'sides' | 'size' | 'center' | 'rotate'>>
```

otherwise you could just define them as numbers and not try to `Pick` them

Reply:
This worked locally, thanks!